pkg/config/config.go

```diff
@@ -38,7 +38,7 @@ type Config struct {
 
 	EnableTelemetry    bool
 	EnableExperimental bool
-	EnableNonRootLogin bool
```

Comment:
After removing this config item, the code in the PD side (PR: https://github.com/tikv/pd/pull/4141/files) doesn't work anymore. So when we submit a new release to PD, we need to remove that line code in PD. Just a reminder.